spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt

```diff
@@ -86,6 +86,7 @@ object SpdxDeclaredLicenseMapping {
         """Apache-2.0 */ &#39; &quot; &#x3D;end --
 
  """.trimIndent() to APACHE_2_0.toExpression(),
+        "Apache-style" to APACHE_2_0.toExpression(),
```

Comment:
Please use a permalink that cannot break in the commit message:

https://github.com/saksdirect/gfc-collection/blob/d65da56/build.sbt#L42

Reply:
Fixed